### PR TITLE
test: Code Coverage: Fy-Header Component

### PR DESCRIPTION
**Pull request by jayfyle**
### Clickup LInk: https://app.clickup.com/t/85zrpv42v

# Edit: Coverage at 100%

fy-header-100

### CODE REVIEW

File: src/app/shared/components/fy-header/fy-header.component.spec.ts [hunk 1]
```diff
@@ -21,4 +24,45 @@ xdescribe('FyHeaderComponent', () => {
...
+    const header = getElementBySelector(fixture, '.fy-header--title');
+    expect(getTextContent(header)).toEqual('Expenses');
+  });
+
+  it('should click a button when HeaderState is SimpleSearch', () => {
+    spyOn(component.simpleSearchCancel, 'emit');
+    component.currentState = HeaderState.simpleSearch;
+    fixture.detectChanges();
+
+    const cancelButton = fixture.debugElement.nativeElement.querySelector('.fy-header--simple-search-cancel');
```
Comment (CODEX108) [inline src/app/shared/components/fy-header/fy-header.component.spec.ts:54]: `const cancelButton= getElementBySelector(fixture, .fy-header--simple-search-cancel') as HTMLElement;`

File: src/app/shared/components/fy-header/fy-header.component.spec.ts [hunk 2]
```diff
@@ -21,4 +24,45 @@ xdescribe('FyHeaderComponent', () => {
...
+    cancelButton.click();
+    expect(component.simpleSearchCancel.emit).toHaveBeenCalledTimes(1);
+  });
+
+  it('should click a button when HeaderState is MultiSelect', () => {
+    spyOn(component.multiselectBack, 'emit');
+    component.currentState = HeaderState.multiselect;
+    fixture.detectChanges();
+
+    const cancelButton = fixture.debugElement.nativeElement.querySelector('ion-button');
```
Comment (CODEX108) [inline src/app/shared/components/fy-header/fy-header.component.spec.ts:64]: same

**Review by CODEX108 — COMMENTED** [review]
Added comments

File: src/app/shared/components/fy-header/fy-header.component.spec.ts [hunk 3]
```diff
@@ -21,4 +24,45 @@ xdescribe('FyHeaderComponent', () => {
...
+    spyOn(component.simpleSearchCancel, 'emit');
+    component.currentState = HeaderState.simpleSearch;
+    fixture.detectChanges();
+
+    const cancelButton = getElementBySelector(fixture, '.fy-header--simple-search-cancel') as HTMLElement;
+    cancelButton.click();
+    expect(component.simpleSearchCancel.emit).toHaveBeenCalledTimes(1);
+  });
+
+  it('should click a button when HeaderState is MultiSelect', () => {
```
Comment (Dimple16) [inline src/app/shared/components/fy-header/fy-header.component.spec.ts:59]: also mention which button?

**Review by Dimple16 — APPROVED** [review]
minor comments, fix and merge

**Pull request merged**